tests/test_util.py

```
@@ -0,0 +1,714 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from datetime import date
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)